### ISSUE TITLE
`sparse-decoration` complains too much

### ISSUE THREAD

**Issue by yegor256** [objectionary/lints#266]
A warning is raised for this code:

```
[] > test
  if. > @
    a
    42
    88
```

It looks perfectly legit to me. The `sparse-decoration` must only complain about this situation:

```
[] > test
  [] > @
    if. > @
      a
      42
      88
```

**Comment by h1alexbel** [comment]
@yegor256 as far I understood, sparse decoration is allowed in application, for instance:

```eo
[] > decorates-application
  if > @
    true
    5
    five
```

should not raise any warnings, right?

**Comment by yegor256** [comment]
@h1alexbel correct

**Comment by maxonfjvipon** [comment]
@yegor256 your example is actually a sparse-decoration. Should be:
```
if. > test
  a
  42
  88
```

**Comment by maxonfjvipon** [comment]
@yegor256 check [this](https://github.com/objectionary/eo/issues/2109#issuecomment-1570416035)

**Comment by maxonfjvipon** [comment]
@yegor256 WDYT?

**Comment by yegor256** [comment]
@maxonfjvipon I agree, my mistake, thanks!
